app/views/admin/financial_reports/index.html.erb

```diff
@@ -1,39 +1,74 @@
+<% content_for :context, "Organisation" %>
+<% content_for :title_margin_bottom, 4 %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body"><%= view_on_website_link_for @organisation, class: "govuk-link" %></p>
+  </div>
 </div>
 
-<%= tab_navigation_for(@organisation) do %>
-  <h2 class="add-bottom-margin">Financial Reports for <%= @organisation.name %></h2>
-  <%= link_to "Add report", {action: 'new'}, {class: 'btn btn-default add-bottom-margin'} %>
-  <% if @organisation.financial_reports.present? %>
-    <table class="table table-bordered table-striped">
-      <thead>
-        <tr class="table-header">
-          <th>Year</th>
-          <th>Funding</th>
-          <th>Spending</th>
-          <th></th>
-          <th></th>
-        </tr>
-      </thead>
-      <tbody>
-        <% @organisation.financial_reports.each do |report| %>
-          <tr>
-            <td><%= report.year %></td>
-            <td><%= number_to_currency(report.funding, precision: 0, unit: '&pound;'.html_safe) %></td>
-            <td><%= number_to_currency(report.spending, precision: 0, unit: '&pound;'.html_safe) %></td>
-            <td><%= link_to 'Edit', edit_admin_organisation_financial_report_path(@organisation, report), method: :get %></td>
-            <td><%= link_to 'Destroy',
-                      admin_organisation_financial_report_path(@organisation, report),
-                      data: { confirm: "Are you sure?" },
-                      method: :delete %></td>
-          </tr>
-        <% end %>
-      </tbody>
-    </table>
-  <% else %>
-    <p class="no-content no-content-bordered">No financial reports</p>
-  <% end %>
-<% end %>
+<%= render "components/secondary_navigation", {
+  aria_label: "Organisation navigation tabs",
+  items: secondary_navigation_tabs_items(@organisation, request.path)
+} %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Financial Reports",
+      margin_bottom: 6
+    } %>
+
+    <%= render "govuk_publishing_components/components/button", {
+      text: "Create new report",
+      href: {action: 'new'},
+      margin_bottom: 6
+    } %>
+
+    <% if @organisation.financial_reports.present? %>
+      <div class="govuk-table--with-actions app-view-financial-reports__listing">
+        <%= render "govuk_publishing_components/components/table", {
+          head: [
+            {
+              text: "Year"
+            },
+            {
+              text: "Funding"
+            },
+            {
+              text: "Spending"
+            },
+            {
+              text: tag.span("Actions", class: "govuk-visually-hidden")
+            }
+          ],
+          rows: @organisation.financial_reports.map do |report|
+            [
+              {
+                text: tag.span(report.year, class: "app-view-financial-reports__listing__year")
```

Comment:
can we just do

```
 text: tag.span(report.year, class: "govuk-!-font-weight-bold")
```

Reply:
> 

Yeah, that would be simpler wouldn't it? 😆